docs/variants/msi_z690/montage.md

```diff
@@ -0,0 +1,123 @@
+# Dasharo compatible with MSI PRO Z690-A DDR4 - montage
```

Comment:
I think that we should change the name of this file, i. e. for something like this: `Presale device assembly and validation`

Reply:
renamed